README.md

```diff
@@ -15,34 +18,144 @@ pitfalls.
 
 ## Getting Started
 
-*   Pre-compiled releases of the compiler are available via:
-    -   [Maven](https://mvnrepository.com/artifact/com.google.javascript/closure-compiler)
-    -   [NPM](https://www.npmjs.com/package/google-closure-compiler) - includes
-        java, native and javascript versions.
-*   See the
-    [Google Developers Site](https://developers.google.com/closure/compiler/docs/gettingstarted_app)
-    for documentation including instructions for running the compiler from the
-    command line.
-
-## Options for Getting Help
-
-1.  Post in the
-    [Closure Compiler Discuss Group](https://groups.google.com/forum/#!forum/closure-compiler-discuss).
-1.  Ask a question on
-    [Stack Overflow](https://stackoverflow.com/questions/tagged/google-closure-compiler).
+The easiest way to get the compiler is to install the precompiled binary
+globally using [NPM](https://npmjs.com) or [Yarn](https://yarnpkg.com):
```

Comment:
Do you want to mention using `npx` to use the compiler without even installing?

Reply:
My reasoning here was to help people get a global binary in their `PATH` as quickly as possible, cross-platform. I believe `npm i -g` and `yarn global add` provide the best way to do this, and this is how I update and interact with the compiler on my own machine.